File: Assets/Scripts/CoinManager.cs
using UnityEngine;
using System.Collections;
using UnityEngine.UI;

public class CoinManager : singleTon<CoinManager>
{
    //public float coins;
    public Text numberCoin;
    public Text timeForNextCoin;
    public CoinSpawner coinSpawner;
    //show the total coins in the guiText
    void Update()
    {
        //numberCoin.text = String.Format("<color=#FF7B40FF><b>Coins: </b></color>" + "<color=#5AFF1AFF>{0}</color>", coins);
        numberCoin.text = Setting.Instance.Coins.ToString();
        timeForNextCoin.text = string.Format("<b>Next in: </b>") + Mathf.Floor(coinSpawner.CoinTimer).ToString() + "s";
    }


}

File: Assets/Scripts/js/CoinManager.js
#pragma strict

var coins: float = 20;
var numberCoin : UI.Text;
var timeForNextCoin : UI.Text;
var coinSpawner : CoinSpawner;
//show the total coins in the guiText
function Update () 
{
       //numberCoin.text = String.Format("<color=#FF7B40FF><b>Coins: </b></color>" + "<color=#5AFF1AFF>{0}</color>", coins);
	   numberCoin.text = coins.ToString();
	   timeForNextCoin.text = String.Format("<b>Next in: </b>") + Mathf.Floor(coinSpawner.CoinTimer).ToString() + "s";
}

